quickwit-search/src/leaf.rs

```diff
@@ -314,6 +319,27 @@ async fn warm_up_terms(searcher: &Searcher, query: &dyn Query) -> anyhow::Result
     Ok(())
 }
 
+async fn warm_up_fieldnorms(searcher: &Searcher, requires_scoring: bool) -> anyhow::Result<()> {
```

Comment:
@PSeitz, is there some optimization we can do here knowing the terms in the query?

Reply:
fieldnorms cost 1 byte per document per field. For a mature split with 5million documents that's 5MB per field. I would consider loading the whole file, since this will probably minimize the costs (we pay per S3 requests).

We have 70ms latency to first byte and around 70Mb/s.
In 210ms total, we can download 2 field norm fields.

Also we can improve the compression for fast fields in tantivy to less than 1byte per doc

Reply:
Thanks, @PSeitz. Thorough answers are always appreciated.